libsolidity/ast/AST.h

```diff
@@ -87,15 +87,13 @@ class ASTNode
 	static void listAccept(std::vector<T> const& _list, ASTVisitor& _visitor)
 	{
 		for (T const& element: _list)
-			if (element)
-				element->accept(_visitor);
+			element->accept(_visitor);
```

Comment:
Should we either keep this or turn it into an assertion?

Reply:
Yeah, good point. Added asserts.